src/Features/Core/Portable/UseNullPropagation/AbstractUseNullPropagationDiagnosticAnalyzer.cs

```diff
@@ -51,13 +51,14 @@ protected AbstractUseNullPropagationDiagnosticAnalyzer()
 
         protected override void InitializeWorker(AnalysisContext context)
         {
+            context.EnableConcurrentExecution();
```

Comment:
this shouldn't be necessary.  AbstractUseNullPropagationDiagnosticAnalyzer is a AbstractCodeStyleDiagnosticAnalyzer, and all of the latter have concurrent execution enabled.

Reply:
Ah, yes, this should be removed. My initial implementation to report cases where analyzers were not concurrent falsely reported analyzers that had compilation start actions.

Reply:
➡️ Fixed